### DISCUSSION TITLE
What's physical margin ?

### DISCUSSION THREAD

**Discussion by daiwanxing** [mdn/mdn-community#542] [Content] [answered]
I read this article [margin-inline-start](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline-start) and encountered a term that is difficult to understand。It's **physical margin**。so please someone can answer me what's  physical margin ?

**Comment by teoli2003** [comment, marked as answer]
The physical margin is the actual margin: top, bottom, right, left.

The logical margin is the margine _relative_ to the text direction: for example margin-start will be the physical margin margin-left for ltr script and margin-right for rtl script. (for example)
